three-tier-main.tf

```diff
@@ -250,7 +252,7 @@ resource "aws_lb" "external-elb" {
 resource "aws_lb_target_group" "external-elb" {
   name     = "ALB-TG"
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure one target group is configured to listen on HTTPS for AWS Load balancer through AWS LB Listener**
 
**Vulnerability**
Not configuring AWS Load balancer to have one target group listening on HTTPS ensures end-to-end encryption is not enabled. This could impact the confidentiality of data in transit.
 
**Remediation**
Elastic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: tenablecs-